plugins/catalog-react/src/alpha/blueprints/extensionData.tsx

```diff
@@ -33,18 +41,50 @@ export const entityFilterExpressionDataRef =
     id: 'catalog.entity-filter-expression',
   });
 
+/** @alpha */
+export type GroupDefinitions = Record<
```

Comment:
```suggestion
export type EntityContentGroupDefinitions = Record<
```

It's so specific and narrowly used, that I thought it could have a specific name. Unless you think it'll be used in more places?